### PR TITLE
Fixes for integration with heroku db

### PR DESCRIPTION
**Pull request by dsayling**

### CODE REVIEW

**Comment by gmemstr** [comment]
Any qualms with using SQLite rather than Postgres? Would keep the footprint small and the sample project contained. @KyleTryon

File: openapi_server/__main__.py [hunk 1]
```diff
@@ -10,17 +11,18 @@
 
 app = connexion.App(__name__, specification_dir="./openapi/")
 app.app.json_encoder = encoder.JSONEncoder
-# app.app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL", "postgresql:///test-cfd")
+logging.info("Looking for DATABASE_URL, if not found an in-memory db will be used")
 app.app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv(
-    "DATABASE_URI", "sqlite:///:memory:"
-)  # can this work if there's no db to connect to?
+    "DATABASE_URL", "sqlite:///:memory:"
```
Comment (dsayling) [inline openapi_server/__main__.py:16]: @gmemstr An in memory sqllite database is used if the database URL is not present

**Review by KyleTryon — APPROVED** [review]
Approving but I need to manually test this locally. Angel and I were working on this separately elsewhere and ran into some issues with flask we may need to put in here.

Will check this ASAP

**Review by killme12993 — APPROVED** [review]

**Review by killme12993 — APPROVED** [review]

**Comment by killme12993** [comment]
I

**Pull request closed without merging**